System.Security.Cryptography/System.Security.Cryptography/AES.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System;
```

Comment:
please add the header

Reply:
If I understand correctly ("add the header"). i done